meeting-notes/2019/Q1/2019-01-07--ipfs-weekly-call.md

```diff
@@ -0,0 +1,76 @@
+# IPFS All Hands Call January 7th, 2019
+
+-   **Moderator:** @pkafei
+-   **Notetaker:** @olizilla
+-   **Attendees:**
+
+    -   _@AddYourNameHere_
+    -   _@prfnv_
+    -   _@jacobheun_
+    -   _@eefahy_
+    -   _@olizilla_
+    -   _@daviddias_
+    -   _@momack2_
+    -   _@jaycarpenter_
+    -   _@vmx_
+    -   _@lidel_
+    -   _@arkadiy_
+    -   _@hugomrdias_
+    -   _@jamiew_
+    -   _@nukemandan_
+    -   _@terichadbourne_
+    -   _@mgoelzer_
+
+  
+**Main Topic:**Actyx: A case study of how one company uses IPFS in their stack.
```

Comment:
@pkafei let me know when the recording is available. I would like to watch the end of the presentation :)

Reply:
Here you go. https://youtu.be/9sWKOaBHHOM

I also put the IPFS Weekly call videos in the[ IPFS Weekly Call Schedule.](https://docs.google.com/spreadsheets/d/1XRB2QsPzCPLPOErKvDZfOKK3CMohI9t_QKNdztYMlK0/edit#gid=350755898)